libs/backend/src/ballot_package/ballot_package_io.ts

```diff
@@ -94,17 +94,15 @@ export async function readBallotPackageFromUsb(
     return filepathResult;
   }
 
-  const artifactAuthenticationResult =
-    await authenticateArtifactUsingSignatureFile({
-      type: 'election_package',
-      filePath: filepathResult.ok(),
-    });
-  if (
-    artifactAuthenticationResult.isErr() &&
-    !isFeatureFlagEnabled(
-      BooleanEnvironmentVariableName.SKIP_BALLOT_PACKAGE_AUTHENTICATION
-    )
-  ) {
+  const artifactAuthenticationResult = isFeatureFlagEnabled(
+    BooleanEnvironmentVariableName.SKIP_BALLOT_PACKAGE_AUTHENTICATION
+  )
+    ? ok()
+    : await authenticateArtifactUsingSignatureFile({
+        type: 'election_package',
+        filePath: filepathResult.ok(),
+      });
+  if (artifactAuthenticationResult.isErr()) {
```

Comment:
Thought it made more sense to actually skip the auth check, instead of still performing the auth check and ignoring the result, when the relevant feature flag is set